server/deskroom/knowledge_base/endpoints.py

```diff
@@ -19,8 +29,59 @@ async def get_knowledge_base(
 ) -> list[KnowledgeBase]:
     response = (
         await supabase.table("knowledge_base")
-        .select("*")
+        .select("*, organizations(company_info_policy)")
         .eq("org_key", item.organization_key)
         .execute()
     )
+
     return response.data
+
+
+@router.post("/upload/{org_key}")
+async def make_knowledge_base(
+    org_key: str,
+    file: UploadFile = File(...),
+    supabase: AsyncClient = Depends(create_supabase_async_client),
+) -> list[KnowledgeBase]:
+    questions = []
+    answers = []
+
+    raw_df = pd.read_excel(
+        file.file.read(), engine="openpyxl", sheet_name="Message data"
+    )
+
+    supabase_response = await (
+        supabase.table("organizations").select("*").eq("key", org_key).execute()
+    )
+    org_info = supabase_response.data[0]
```

Comment:
index error가 나기 쉬운 코드로 보입니다. 
```suggestion
    org_info = next(supabase_response.data, None)
    if not org_info:
        raise SomeError("some error description")
```
요렇게 하는 게 더 안전하지 않을까요? 클라이언트 입장에서는 500만 뜰거라서 무슨 에러가 나는 지 알수가 없어서요.

Reply:
음... 이걸 사용하는게 org_id랑 name_eng밖에 없는데 해당하는 key값이 테이블안에 있다면 id랑 name도 있을테니 supabase table query할때 error 체크하게 바꿔보겠습니다